gen/flights/flight.py

```diff
@@ -137,7 +137,8 @@ class Flight:
     def __init__(self, package: Package, unit_type: Type[FlyingType],
                  count: int, flight_type: FlightType, start_type: str,
                  departure: ControlPoint, arrival: ControlPoint,
-                 divert: Optional[ControlPoint]) -> None:
+                 divert: Optional[ControlPoint],
+                 custom_name: Optional[str] = None) -> None:
```

Comment:
What's the difference between `None` and an empty name? Doesn't seem like there is one. A signal that the name wasn't chosen seems to be the idea, but if the player types something and then deletes it it looks like that gives the empty string instead of `None`?

Reply:
I just tested this, and the behavior is correct with either `None` or an empty string.